.github/workflows/deploy.yml

```diff
@@ -50,4 +50,5 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./docs/.vitepress/dist
           enable_jekyll: false
+          force_orphan: true
```

Comment:
Why was this needed?

Reply:
Sorry, forgot to leave a comment about this. I think it just makes `gh-pages` branch a one-commit branch, which I think makes more sense because it's just basically a copy of whatever is in the `dist` folder.